.docker-mongo/entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+mongod --fork --config /etc/mongod.conf
```

Comment:
even though not meant for production and only preview... Maybe we should add: `--smallfiles --storageEngine=mmapv1`

For sure I think the storageEngine bit.